docs/src/configuration/services/elasticsearch.md

```diff
@@ -32,6 +32,18 @@ To switch to OpenSearch, follow the same procedure as for [upgrading](#upgrading
 Elasticsearch 7.9 isn't available in the EU-1 and US-1 regions.
 Consider [region migration](../../guides/general/region-migration.md) if your project is in those regions.
 
+## Use a framework
+
+If you use one of the frameworks, follow the guide there:
+
+- [Drupal 9](../../guides/drupal9/elasticsearch.md)
+- [Micronaut](../../guides/micronaut/elasticsearch.md)
+- [Quarkus](../../guides/quarkus/elasticsearch.md)
+- [Jakarta EE](../../frameworks/jakarta.md#elasticsearch)
```

Comment:
I'd keep alphabetical order. Or, I'm not sure why Jakarta is here in the order?

Similarly with the other instances below.